codegen/casing.go

```diff
@@ -29,7 +29,10 @@ import (
 	"unicode/utf8"
 )
 
-var pascalCaseMap *sync.Map
+var (
+	pascalCaseMap *sync.Map
```

Comment:
Why not just use `pascalCaseMap sync.Map`, which will create a sync.Map and we don't need sync.Once and no need to check for nil.